runtimes/manta/src/command.rs

```diff
@@ -167,3 +150,25 @@ pub fn run() -> sc_cli::Result<()> {
 		}
 	}
 }
+
```

Comment:
who calls the run() method here ?

Reply:
@ghzlatarev  It is kind of a Client hook that the Client library code call.